velox/common/fuzzer/Utils.h

```diff
@@ -40,20 +40,29 @@ enum UTF8CharList {
 
 bool coinToss(FuzzerGenerator& rng, double threshold);
 
+/// Generate a random type with given scalar types. The level of nesting is up
+/// to maxDepth. If keyTypes is non-empty, choosing from keyTypes when
+/// determining the types of map keys. If keyTypes is empty, choosing from
+/// scalarTypes for the types of map keys.
```

Comment:
nit: since we are passing a reference for types `std::vector<TypePtr>&` lets just change the API to accept keyTypes and ValueTypes to make the API more uniform.